docs/en/latest/FAQ.md

```diff
@@ -594,6 +594,36 @@ The differences between the two are described in the table below:
 | Used when there are property changes that needs to be propagated across all configuration instances of a Plugin. | Used when you need to reuse a common set of configuration instances so that it can be extracted to a `plugin-config` and bound to different Routes. |
 | Takes effect on all the entities bound to the configuration instances of the Plugin.                             | Takes effect on Routes bound to the `plugin-config`.                                                                                                |
 
+## After deploying Apache APISIX, how to detect the survival of the APISIX data plane?
```

Comment:
👋 I just did a quick search, but I'm still unsure how to express the meaning of whether API Gateway is running or in trouble. Any ideas? @moonming